src/SourcemapToolkit.SourcemapParser/SourceMapGenerator.cs

```diff
@@ -6,6 +6,9 @@
 
 namespace SourcemapToolkit.SourcemapParser
 {
+	using System.Linq;
```

Comment:
Same thing about putting the imports next to the other imports in this file instead of inside the namespace

Reply:
+